### PR TITLE
it: customization autodiff translation

### PR DESCRIPTION
**Pull request by mcaci**
Hello,

I open this PR to add a new page for the Italian translation of the docs.

This one is about the content of the file `customization\autodiff.ipynb`

Cheers,
Michele

### CODE REVIEW

File: site/it/tutorials/customization/autodiff.ipynb [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "4U1KKzUpNl58"
+      },
+      "source": [
+        "## Passi successivi\n",
+        "\n",
+        "In questo tutorial abbiamo coperto il calcolo di gradienti in TensorFlow. Con questo abbiamo abbastanza primitive richieste per costruire e allenare reti neurali."
```
Comment (bhack) [inline site/it/tutorials/customization/autodiff.ipynb]: Addestrare

**Comment by mcaci** [comment]
I think I will redo the PR. The syncing did not go properly

**Comment by lamberta** [comment]
I rebased and (force) pushed to your branch. Should be ok---but make sure you are working on the latest before adding new commits

**Comment by mcaci** [comment]
Thank you @lamberta! You are right I noticed too late that I didn't have an updated repo. Also I have to practise with the interactive rebase and force push to clean commit history next time

File: site/it/tutorials/customization/autodiff.ipynb [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "j8BT7T8yf6Rv"
+      },
+      "outputs": [],
+      "source": [
```
Comment (lamberta) [inline site/it/tutorials/customization/autodiff.ipynb]: Remove empty trailing cell

Comment (mcaci) [reply site/it/tutorials/customization/autodiff.ipynb]: It should be removed now

File: site/it/tutorials/customization/autodiff.ipynb [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+        "  \u003c/td\u003e\n",
+        "\u003c/table\u003e"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "vDJ4XzMqodTy"
+      },
```
[the remark this answers is not in the document]
Comment (mcaci) [reply site/it/tutorials/customization/autodiff.ipynb]: New text cell added

File: site/it/tutorials/customization/autodiff.ipynb [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+        "colab_type": "text",
+        "id": "idv0bPeCp325"
+      },
+      "source": [
+        "\u003ctable class=\"tfo-notebook-buttons\" align=\"left\"\u003e\n",
+        "  \u003ctd\u003e\n",
+        "    \u003ca target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/customization/autodiff\"\u003e\u003cimg src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" /\u003eVisualizza su TensorFlow.org\u003c/a\u003e\n",
+        "  \u003c/td\u003e\n",
+        "  \u003ctd\u003e\n",
+        "    \u003ca target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/customization/autodiff.ipynb\"\u003e\u003cimg src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" /\u003eEsegui in Google Colab\u003c/a\u003e\n",
```
[the remark this answers is not in the document]
Comment (mcaci) [reply site/it/tutorials/customization/autodiff.ipynb]: All 3 references to "en/" are now to "it/"

**Comment by mcaci** [comment]
Thank you @lamberta and thanks for the suggestion, I have added a JSON linter extension to my vs code but I take also the jsonlint validator just in case

**Pull request merged**